### ISSUE TITLE
Large component library app investigations

### ISSUE THREAD

**Issue by NTaylorMullen** [dotnet/aspnetcore#31357]
- [ ] Telerik
- [ ] DevExpress
- [ ] Radzen

@danroth27 can you add more here? Also, if we have contacts it'd be great to get large app examples for us to dig into.

**Comment by NTaylorMullen** [comment]
@ryanbrandenburg when we get to this we'll need to pick a subset of the ones in the component-bundles that Dan mentioned and give them a shot. Also, Dan has been sending me all sorts of apps to test against that use these so that can also be an influencer

**Comment by NTaylorMullen** [comment]
I'm digging into Telerik here.

**Comment by NTaylorMullen** [comment]
Findings:
- https://github.com/dotnet/aspnetcore/issues/31939

**Comment by ryanbrandenburg** [comment]
Hmm, the only problems I could see to stick out were already tracked by https://github.com/dotnet/aspnetcore/issues/32118 and [this issue](https://developercommunity.visualstudio.com/t/ServiceHubRoslynCodeAnalysisServiceex/1359690). Not sure if you found and fixed everything already or I'm just not being very inventive. Will come back to this later, hopefully my creativity will recover.

A note on https://github.com/dotnet/aspnetcore/issues/32118, I tried to do a fix for the string allocation problems I found, but it didn't seem to actually move the needle on the benchmark, so I'll leave the fix to the Razor Compiler team who knows better.
